[2022-12-11 19:22:45,269] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-09T15:00:00+00:00 [queued]>
[2022-12-11 19:22:45,294] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-09T15:00:00+00:00 [queued]>
[2022-12-11 19:22:45,296] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:22:45,298] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-12-11 19:22:45,299] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 19:22:45,357] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_etl_process> on 2022-12-09 15:00:00+00:00
[2022-12-11 19:22:45,367] {standard_task_runner.py:52} INFO - Started process 396 to run task
[2022-12-11 19:22:45,377] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_DAG', 'run_etl_process', 'scheduled__2022-12-09T15:00:00+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/etl_DAG.py', '--cfg-path', '/tmp/tmpy0xhpnla', '--error-file', '/tmp/tmpcmmhgmj2']
[2022-12-11 19:22:45,380] {standard_task_runner.py:80} INFO - Job 232: Subtask run_etl_process
[2022-12-11 19:22:45,558] {task_command.py:369} INFO - Running <TaskInstance: etl_DAG.run_etl_process scheduled__2022-12-09T15:00:00+00:00 [running]> on host 0163e0790208
[2022-12-11 19:22:45,875] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_DAG
AIRFLOW_CTX_TASK_ID=run_etl_process
AIRFLOW_CTX_EXECUTION_DATE=2022-12-09T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-09T15:00:00+00:00
[2022-12-11 19:22:51,537] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 19:22:51,560] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_DAG, task_id=run_etl_process, execution_date=20221209T150000, start_date=20221211T192245, end_date=20221211T192251
[2022-12-11 19:22:51,622] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 19:22:51,685] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
